spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1183,6 +1184,9 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
   public InterpreterResult interpret(String[] lines, InterpreterContext context) {
     synchronized (this) {
       z.setGui(context.getGui());
+      String paraText = context.getParagraphText();
+      sc.setCallSite(new CallSite(Utils.getJobShortText(paraText),
```

Comment:
hmm, then it is fine not do it for SparkR for now. Could you create a spark ticket for that ?